receiver/splunkenterprisereceiver/metadata.yaml

```diff
@@ -239,3 +239,5 @@ metrics:
       value_type: int
     attributes: [splunk.queue.name] 
 
+tests:
+  config:
```

Comment:
Why do we need this?

Reply:
they added the test but not the metadata.yaml snippet, so the code was not impacted by the change.